internal-registry/che-incubator/command-line-terminal/4.5.0/meta.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v2
+publisher: che-incubator
+name: command-line-terminal
+version: 4.5.0
+type: Che Editor
+displayName: Command Line Terminal
+title: Command Line Terminal
+description: Command Line Terminal provides the ability to start a terminal inside the OpenShift Console.
```

Comment:
does this also work with k8s? Or is it exclusively targetting OpenShift?

Reply:
It exclusively targeting OpenShift.
If be more precisely, it provides an API for OpenShift Console to initialize container before opening terminal (create kubeconfig).
There is already API to cache terminal sessions, but OpenShift Console does not use it yet.

Possible Kubernetes Dashboard could be improved to use it as well but nobody's taken a look at that.
So, at this point, we declare only OpenShift Terminal Providing.

Reply:
works for me, just making sure your description was accurate since most of Che features are for BOTH k8s & OS.

Reply:
Can we please update with name "OpenShift Terminal" - short and sweet - instead of "Command Line Terminal"?

Reply:
@pdaverh are we changing the name again? We aligned to [CRW-600](https://issues.redhat.com/browse/CRW-600). We are avoiding to prefix it with OpenShift since it's not an OpenShift product (from @nickboldt comment above "You need Brand/Marketing/PM approval to use the name in anything producty/projecty like this"). Moreover this is the name of a `cheEditor`, not the name of the Operator hence I don't think it really matters because it won't be exposed to users.

@sleshchenko I would merge this PR like this. @pdaverh we can later update this cheEditor meta.yaml if we all agree on a better naming. We may need a naming dedicated call with @nickboldt.

Reply:
Thanks for the information that the name here will not be exposed to users. For the operator, we are working with brand and marketing to approve "OpenShift Terminal". OpenShift PM is fine with it.